TYPO3.Media/Documentation/AsynchronousThumbnails.rst

```diff
@@ -38,4 +38,4 @@ can experience performance issues. Therefore it is recommended to configure the
 
   Configure ``crontab`` to run the render command every minute: ``* * * * * /path/to/site/flow media:renderthumbnails``
 
-  Use ``media:clearthumbnails`` and ``media:createthumbnails`` to refresh thumbnails.
\ No newline at end of file
+  Use ``media:clearthumbnails`` and ``media:createthumbnails`` to refresh thumbnails.
```

Comment:
The documentation will point to a non-existing setting (for now), so don't merge without actual vendor name change.